From fc48f4851829b99be48416189750cc7d201548a6 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Sun, 7 Apr 2019 21:26:11 +0300
Subject: [PATCH 140/386] arm64: dts: cn913x: disable SFP for CP0 eth0 and CP2
 eth0

Similar to
commit 68c4e1facb74 ("arm64: dts: cn9131: disable SFP for CP1 eth0")
Recently enabled CONFIG_SFP unveiled a problem with CP0 eth0 SFP cage
connectivity caused fails on DPDK tests. Apply work-around
re-enabling normal functionality by temporary disabling the SFP node
in device tree.

Change-Id: Id18af3d737d67704fec30b2f6c76196fdcbcb461
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/7071
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Marcin Wojtas <Marcin.Wojtas@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-db.dtsi | 1 +
 arch/arm64/boot/dts/marvell/cn9132-db.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
index ea517dc04c99..003b1b364d5f 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
@@ -66,6 +66,7 @@
 		mod-def0-gpio = <&cp0_moudle_expander1 10 GPIO_ACTIVE_LOW>;
 		tx-disable-gpio = <&cp0_moudle_expander1 9 GPIO_ACTIVE_HIGH>;
 		tx-fault-gpio = <&cp0_moudle_expander1 8 GPIO_ACTIVE_HIGH>;
+		status = "disabled";
 	};
 };
 
diff --git a/arch/arm64/boot/dts/marvell/cn9132-db.dtsi b/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
index e6c43dd7c5cc..eb1ac000208e 100644
--- a/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9132-db.dtsi
@@ -69,6 +69,7 @@
 		mod-def0-gpio = <&cp2_moudle_expander1 10 GPIO_ACTIVE_LOW>;
 		tx-disable-gpio = <&cp2_moudle_expander1 9 GPIO_ACTIVE_HIGH>;
 		tx-fault-gpio = <&cp2_moudle_expander1 8 GPIO_ACTIVE_HIGH>;
+		status = "disabled";
 	};
 };
 
-- 
2.17.1

